test/common/run-tests

```diff
@@ -143,12 +145,9 @@ class Test:
     # internal methods
 
     def __str__(self):
-        return "{0} {1} {2}{3}".format(
-            self.test_id,
-            self.command[0],
-            self.command[-1],
-            " [ND@{0}]".format(self.machine_id) if self.nondestructive else "",
-        )
+        cost = "" if self.cost == 1 else f" 🏋 {self.cost}"
```

Comment:
This is mostly playing around, and at least it will help the first test run to validate correct costs. If that leads to problems with the HTML log parser, naughties, etc. I'm happy to just take this out again.

Reply:
Replaced with a simple `$`, as even in 2022 Firefox still defaults to something non-UTF8 for plain text files.